src/Select/Select.js

```diff
@@ -62,16 +64,16 @@ const Select = React.forwardRef(({
             {...props}
             className={selectClasses}
             id={id}
-            onClick={handleClick}
+            onClick={disabled || readOnly ? null : handleClick}
```

Comment:
@SAP/fundamental-react-team Should the dropdown still open if it's readonly? The spec doesn't specify and I know native `<select>` cannot be readOnly, so there's not a lot to go on here